src/G2/Initialization/MkCurrExpr.hs

```diff
@@ -68,35 +64,6 @@ mkMainExpr tc kv ng ex =
     in
     (app_ex, is, typsE, ng')
 
--- | This implementation aims to symbolically execute functions 
--- treating both types and value level argument as symbolic
-mkMainExprNoInstantiateTypes :: Expr -> NameGen -> (Expr, [Id], [Expr], NameGen)
```

Comment:
It looks like for some reason a bunch of stuff is getting cut that shouldn't be?  Any idea what is going on here?  (Applies to multiple places in the code, not just this function.)

Reply:
I did manually revert the changes to symbolic function templates that were causing issues for Nebula.  Maybe this was a side effect of that.

Reply:
Ok, let's unrevert them before merging this in?  Would you be able to put in a bug report of what had to be reverted though, so that any issues can be fixed at some point?